libkbfs/test_stallers.go

```diff
@@ -38,6 +39,166 @@ const (
 	StallableMDPutUnmerged           StallableMDOp = "PutUnmerged"
 )
 
+type naïveStallContext struct {
```

Comment:
I agree with @strib in that Context might not be the best word. `naiveStall` works, or maybe `naiveStallInfo`.